lld/ELF/Driver.cpp

```diff
@@ -560,6 +564,25 @@ static uint8_t getZStartStopVisibility(opt::InputArgList &args) {
   return ret;
 }
 
+static StringRef getZGcs(opt::InputArgList &args) {
+  StringRef ret = "implicit";
+  for (auto *arg : args.filtered(OPT_z)) {
+    std::pair<StringRef, StringRef> kv = StringRef(arg->getValue()).split('=');
+    if (kv.first == "gcs") {
+      arg->claim();
+      if (kv.second == "implicit" || kv.second == "always" ||
```

Comment:
We shall define a `GcsPolicy`. I believe we can reuse it or will soon need it for other things.